src/main/java/io/spokestack/spokestack/profile/VADTriggerKeywordASR.java

```diff
@@ -0,0 +1,56 @@
+package io.spokestack.spokestack.profile;
+
+import io.spokestack.spokestack.PipelineProfile;
+import io.spokestack.spokestack.SpeechPipeline;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A speech pipeline profile that uses voice activity detection to activate
+ * ASR based on keyword detection.
```

Comment:
I keep getting confused in my head. Keyword detection is an ASR event, or a wakeword-style pipeline activation event?

Reply:
The keyword recognizer fires `RECOGNIZE` (ASR) events; this profile uses the VAD trigger to fire `ACTIVATE`.